_episodes/23-continuous-integration-automated-testing.md

```diff
@@ -56,22 +56,22 @@ Once complete, it presents a report to let you see what happened.
 There are many CI infrastructures and services,
 free and paid for,
 and subject to change as they evolve their features.
-We will be looking at [GitHub Actions](https://github.com/features/actions) -
-which unsurprisingly is available as part of GitHub.
+We will be looking at [GitLab CI/CD](https://docs.gitlab.com/ee/ci/) -
```

Comment:
The link points to the GitLab EE (Enterprise Edition) documentation. I think it does not make a difference here, but perhaps the general introduction is more suitable?
```suggestion
We will be looking at [GitLab CI/CD](https://about.gitlab.com/topics/ci-cd/) -
```
I am not entirely sure through, because the link I am pointing is less technical.

Reply:
I think it does not really matter. I think participants will be used to the enterprise edition actually. But we will use your link!